src/Presenters/ApiPresenter.php

```diff
@@ -156,8 +150,9 @@ private function getApi(Request $request): Api
     private function checkAuth(ApiAuthorizationInterface $authorization): ?IResponse
     {
         if (!$authorization->authorized()) {
```

Comment:
toto by sme potrebovali zabalit este do toho handleAuthorization. Use case:
nas authorizacny servis hodi exception, ktoru potrebujeme pretransformovat na API error response, to tu neurobime. Ale ak by to volanie bolo vnutri handleAuthorization, vieme spravit nejaky try catch a do responsu hodit json response s chybou

Reply:
Velmi sa mi nepacilo checkovat $authorization->authorized() v error handleri. 
Tak som dal try catch von a pridal metodu na handlovanie AuthorizationException.

Reply:
Hej ani mne. Spravime to tak, že ti dám aj ten nadväzujúci task a priprav to cele tak, aby to fungovalo